plutip.cabal

```diff
@@ -130,7 +130,8 @@ library
   ghc-options:
     -Wall -Wcompat -Wincomplete-uni-patterns -Wredundant-constraints
     -Wmissing-export-lists -Wmissing-deriving-strategies
-    -Wmissing-import-lists -Werror
+    -Wmissing-import-lists 
+    -- -Werror
```

Comment:
I saw @zmrocze's suggestion to re-enable this (which is a good default for applications). It can be kind of annoying during development though, so maybe it's worth it to add a `+production` flag where this is enabled and otherwise disabled by default?

Reply:
Yeah, I'm also all for either `production` flag for enabling, or `development` for disabling. Will make an issue.